### DISCUSSION TITLE
Some type errors during install eggs cli

### DISCUSSION THREAD

**Discussion by lengfangbing** [nestdotland/nest.land#303] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/46396510/98112537-56ee5e00-1edd-11eb-884e-9ccbbcbfa408.png)
![image](https://user-images.githubusercontent.com/46396510/98112561-62418980-1edd-11eb-9f98-47071903d476.png)
could someone help me? :(

**Comment by maximousblk** [comment, marked as answer]
This was a deno bug. Upgrade to the latest deno (1.5.1) and try again. Also install the latest eggs (0.3.2)

**Reply by lengfangbing** [reply]
Thanks a lot 👍
